### ISSUE TITLE
Benchmarks are doing different things. 

### ISSUE THREAD

**Issue by v3ss0n** [channelcat/sanic#207]
If you want a legitimate benchmark  , they all should do the same thing:

```python
import ujson
from tornado import ioloop, web


class MainHandler(web.RequestHandler):
    def get(self):
        self.write(ujson.dumps({'test': True}))


app = web.Application([
    (r'/', MainHandler)
],  debug=False,
    compress_response=False,
    static_hash_cache=True
)

app.listen(8000)
ioloop.IOLoop.current().start()
```
Tornado is using ujson and dumping json 

compare to golang which is just doing print

```
package main

import (
    "fmt"
    "os"
    "net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
    fmt.Fprintf(w, "Hi there, I love %s!", r.URL.Path[1:])
}

func main() {
    http.HandleFunc("/", handler)
    http.ListenAndServe(":" + os.Args[1], nil)
}
```

**Comment by kgantsov** [comment]
I've created pull request which is fixing golang performance test https://github.com/channelcat/sanic/pull/220

**Comment by seemethere** [comment]
Closed with #220
